docs/sources/articles/runmetrics.md

```diff
@@ -113,7 +113,7 @@ indicates the number of page faults which happened since the creation of
 the cgroup; this number can never decrease).
 
 
- - **cache:**  
+ - **cache:**
```

Comment:
I think these are intended in this document, just by looking at the source and the current live docs; https://docs.docker.com/articles/runmetrics/